traffic_router build failure in master

Latest master build failed in Jenkins: https://builds.apache.org/view/S-Z/view/TrafficControl/job/trafficcontrol-master-build/997/console

```
traffic_router_build_1   | Initializing the build area.
traffic_router_build_1   | ./build_rpm.sh: line 57: wget: command not found
traffic_router_build_1   | The build area has been initialized.
traffic_router_build_1   | Building the rpm for el7.
traffic_router_build_1   | error: File /tmp/trafficcontrol/rpmbuild/SOURCES/apache-tomcat-8.5.32.tar.gz: No such file or directory
traffic_router_build_1   | RPM BUILD FAILED: 1
traffic_router_build_1   | traffic_router failed: 
traffic_router_build_1   | The following subdirectories had errors: 
traffic_router_build_1   |    traffic_router
traffic_router_build_1   | + echo 1
traffic_router_build_1   | + exit 1
```

--- a/infrastructure/docker/build/Dockerfile-traffic_router
+++ b/infrastructure/docker/build/Dockerfile-traffic_router
@@ -32,7 +32,7 @@ RUN	yum -y install \
 ### traffic_router specific requirements
 RUN	yum -y install \
 		which  \
-                wget && \
+                curl && \
 	yum -y clean all
 
 

--- a/traffic_router/build/build_rpm.sh
+++ b/traffic_router/build/build_rpm.sh
@@ -38,7 +38,8 @@ function installDnsSec {
 	local dnssec="$dnssectools-$dnssecversion"
 	local dnssecurl=http://www.verisignlabs.com/dnssec-tools/packages/old-releases
 
-	curl -o "$dnssec".tar.gz "$dnssecurl/$dnssec".tar.gz || \
+	echo "Downloading $dnssec library..."
+	curl -fo "$dnssec".tar.gz "$dnssecurl/$dnssec".tar.gz || \
 		{ echo "Could not download required $dnssec library: $?"; exit 1; }
 	tar xzvf "$dnssec".tar.gz ||  \
 		{ echo "Could not extract required $dnssec library: $?"; exit 1; }

--- a/traffic_router/tomcat-rpm/build_rpm.sh
+++ b/traffic_router/tomcat-rpm/build_rpm.sh
@@ -54,7 +54,9 @@ function initBuildArea() {
         export VERSION=$TOMCAT_VERSION
         export RELEASE=$TOMCAT_RELEASE
 
-        wget http://archive.apache.org/dist/tomcat/tomcat-8/v$VERSION.$RELEASE/bin/apache-tomcat-$VERSION.$RELEASE.tar.gz -O "$RPMBUILD"/SOURCES/apache-tomcat-$VERSION.$RELEASE.tar.gz
+        echo "Downloading Tomcat $VERSION.$RELEASE..."
+        curl -fo "$RPMBUILD"/SOURCES/apache-tomcat-$VERSION.$RELEASE.tar.gz http://archive.apache.org/dist/tomcat/tomcat-8/v$VERSION.$RELEASE/bin/apache-tomcat-$VERSION.$RELEASE.tar.gz || \
+        { echo "Could not download Tomcat $VERSION.$RELEASE: $?"; exit 1; }
 
         cp "$TR_DIR/tomcat-rpm/tomcat.service" "$RPMBUILD/SOURCES/" || { echo "Could not copy source files: $?"; exit 1; }
         cp "$TR_DIR/tomcat-rpm/tomcat.spec" "$RPMBUILD/SPECS/" || { echo "Could not copy spec files: $?"; exit 1; }

